apps/admin-panel/src/App.jsx

```diff
@@ -1672,6 +1672,14 @@ const TpCompanyProfileShow = (props) => (
           <TextField source="linkedInUrl" />
           <TextField source="phoneNumber" />
           <TextField source="about" />
+          <TextField
+            label="How They Heard about ReDI Talent Pool"
+            source="howDidHearAboutRedi"
+          />
+          <TextField
```

Comment:
Nice stuff. If you wanna take it one step further and display this conditionally, you can access the `TpCompanyProfile` instance through `props.record`

Reply:
Should we show it conditionally? Since it's gonna be required, it's gonna exist for all new companies anyway.

Reply:
Yes, makes sense to show this in any case. It's the next field that should be shown conditionally (i.e. if reason = other).